src/scripts/modules/media/body/body.coffee

```diff
@@ -152,6 +152,31 @@ define (require) ->
             # NOTE: `.toggleClass()` explicitly requires a `false` (not falsy) 2nd argument
             $el.toggleClass('ui-has-child-title', $title.length > 0)
 
+          # Create subtitles in each chapter with corresponding ids as their href
+          total_subtitle = $temp.find("[data-depth='1']")
+          total_id = total_subtitle.children('[data-type="title"]')
+          id_length = total_id.length
+          i = 0
+          slash_index = window.location.href.lastIndexOf("/")+1
+          real_href = window.location.pathname
+          if real_href.indexOf("#")!= -1
+            hash_index = real_href.indexOf("#")-1
+            real_href = real_href.substring(0,hash_index)
+          location = $('.table-of-contents [href$="'+real_href+'"]').children(".title")
+          location2 = $('.table-of-contents [href$="'+real_href+'"]').children(".chapter-number")
+          # Keep adding subtitles until all subtitles are added
+          if location.children("li").length < id_length
+            while i < id_length
+              c_index = i + 1
+              id = total_subtitle[i].id
+              hash_id = "#" + id
+              list = $("<li>")
+              list.text(location2[0].textContent + "."  + c_index + " " + total_id[i].textContent)
```

Comment:
I think this number is incorrect. It seems to be `{CHAPTER}.{SECTION_IN_THIS_PAGE}` which is misleading because students are probably expecting the number to be `{CHAPTER}.{SECTION_AKA_PAGE}.{SECTION_IN_THIS_PAGE}`

Is this numbering necessary or would it make sense to just number the sections in order, like https://en.wikipedia.org/wiki/OpenStax_College for example?

If this is intended to be a temporary solution then please ignore the following... I think this may need to be internationalized. For example, the `.` separator character means something different in English than it does in other languages (like Polish). That may be one reason why Wikipedia just numbers the sections

/cc @edwoodward